### DISCUSSION TITLE
v0.20.3

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1495] [Releases]
> [!WARNING]  
> `v0.20.x` has breaking changes and requires action. Read the original release notes here: https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0

## What's Added
* [Feature] Use ping instead of a HTTP get request to check for internet connection by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1493

## What's Changed
* [Chore Round package loss in telegram notification by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1491
* [Bug] Fixed result reference in speedtest completed mail by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1492
* Release v0.20.3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1494

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.20.2...v0.20.3

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.3'>v0.20.3</a>.</em>

**Comment by edd080** [comment]
Thank you so much for this update, email reporting is working again, thank you so much for your work.👍🏻😁
